docs/reference/generated/menu-popup.json

```diff
@@ -18,6 +18,10 @@
     "data-closed": {
       "description": "Present when the menu is closed."
     },
+    "data-instant": {
+      "description": "Indicates the instant type of the menu popup.",
```

Comment:
TBH this description doesn't help much to understand what this attribute is for.

Reply:
Yep, agreed. The one for Popover is the same but I can update both of them.